### PR TITLE
Add UI support for concertContact

### LINKED ISSUE #97: Add Ui support for ConcertContact

### PR DESCRIPTION
**Pull request by hj235**
I put a "Show CC" button in the menu bar at the top for now, once we get the list command for CC working can consider removing it.

Resolve #97

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -78,6 +91,8 @@ public Stage getPrimaryStage() {
 
     private void setAccelerators() {
         setAccelerator(helpMenuItem, KeyCombination.valueOf("F1"));
+        // breaks the program somehow???
+        // setAccelerator(toggleConcertContactVisibilityItem, KeyCombination.valueOf("Ctrl+Tab"));
```
Comment (hj235) [inline src/main/java/seedu/address/ui/MainWindow.java]: any1 know y

Comment (wes-w-z-h) [reply src/main/java/seedu/address/ui/MainWindow.java]: did you try setting different KeyCombinations instead of ctrl+tab like ctrl+[some other key]

Comment (hj235) [reply src/main/java/seedu/address/ui/MainWindow.java]: Yeah Ctrl+Tab also worked if i set it for the Help function. But it crashes for the show cc for some reason no matter what i do with it. Disabling the help function also didnt work. Without setting the key combi the show cc looks like it works perfectly fine tho.

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -42,12 +43,24 @@ public class MainWindow extends UiPart<Stage> {
     @FXML
     private MenuItem helpMenuItem;
 
+    @FXML
+    private MenuItem toggleConcertContactVisibilityItem;
```
Comment (bbbbcccd) [inline src/main/java/seedu/address/ui/MainWindow.java]: change this to match the id in fxml (toggleConcertContactViewItem)

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (bbbbcccd) [reply src/main/java/seedu/address/ui/MainWindow.java]: toggleConcertContactVisibilityItem -> toggleConcertContactViewItem

**Review by bbbbcccd — CHANGES_REQUESTED** [review]
ui looks good, make the changes I commented and it should work

**Review by bbbbcccd — APPROVED** [review]
lgtm! Well done UI expert! 💯 🥇

**Pull request merged**